### PR TITLE
Add lots - but not all - proto messages for analyzers to python-proto

### PR DESCRIPTION
**Pull request by wimax-grapl**
### Which issue does this PR correspond to?
Precursor work for https://github.com/grapl-security/issue-tracker/issues/918

### What changes does this PR make to Grapl? Why?
Add serde classes for most things in analyzer, with the exception of a GraphView. 

### How were these changes tested?
unit testing

### CODE REVIEW

File: src/python/python-proto/python_proto/api/plugin_sdk/analyzers/v1beta1/messages.py [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+            case StringPropertyUpdate() as inner:
+                msg.string_property.CopyFrom(inner.into_proto())
+            case UInt64PropertyUpdate() as inner:
+                msg.uint64_property.CopyFrom(inner.into_proto())
+            case Int64PropertyUpdate() as inner:
+                msg.int64_property.CopyFrom(inner.into_proto())
+            case EdgeUpdate() as inner:
+                msg.edge.CopyFrom(inner.into_proto())
+            case _:
+                raise Exception(f"Unknown variant: {self.inner}")
```
Comment (colin-grapl) [inline src/python/python-proto/python_proto/api/plugin_sdk/analyzers/v1beta1/messages.py:193]: Isn't this exhaustive?

Comment (wimax-grapl) [reply src/python/python-proto/python_proto/api/plugin_sdk/analyzers/v1beta1/messages.py:193]: https://peps.python.org/pep-0622/#check-exhaustiveness-at-runtime

i wouldn't expect this to ever happen, in your PR it's `assert_never`

File: src/python/python-proto/python_proto/tests/test_grapl_common.py [hunk 2]
```diff
@@ -9,9 +9,7 @@
 
 
 def uids(
-    value: st.SearchStrategy[int] = st.integers(
-        min_value=strategies.UINT64_MIN, max_value=strategies.UINT64_MAX
-    ),
+    value: st.SearchStrategy[int] = strategies.uint64s,
```
Comment (colin-grapl) [inline src/python/python-proto/python_proto/tests/test_grapl_common.py:12]: uids can not be 0

**Review by colin-grapl — APPROVED** [review]

File: src/python/python-proto/python_proto/api/plugin_sdk/analyzers/v1beta1/messages.py (hunk 1 above)
Comment (colin-grapl) [reply src/python/python-proto/python_proto/api/plugin_sdk/analyzers/v1beta1/messages.py:193]: ya assert_never is mypy time tho, nbd

**Pull request merged**